.gitignore

```diff
@@ -25,3 +25,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+#cypress
+.env.dev
```

Comment:
Please use existed .env.development. It's allowed to add new env variables

Reply:
There are login and password in .env.dev which is in .gitignore

Reply:
This local env file is in git ignore `.env.development.local`

Reply:
fixed